src/hotspot/share/gc/serial/cardTableRS.cpp

```diff
@@ -363,13 +361,13 @@ CardTable::CardValue* CardTableRS::find_first_dirty_card(CardValue* const start_
   }
 
   // Word comparison
-  while (current_card + sizeof(Word) <= end_card) {
-    Word* current_word = reinterpret_cast<Word*>(current_card);
-    if (*current_word != (Word)clean_card_row_val()) {
+  while (current_card + sizeof(CardWord) <= end_card) {
+    CardWord* current_word = reinterpret_cast<CardWord*>(current_card);
```

Comment:
When the idea of something like CardWord was suggested, a named accessor to package this was part
of that suggestion.

Reply:
I find that too "caller-specific" to be included inside `CardTable`. Creating an abstraction here breaks the flow, so I kept it as is.

Reply:
CardTable is providing CardWord and is_word_aligned (or whatever it gets called), so a function for
obtaining a CardWord from the CardTable seems entirely reasonable.

Reply:
In this context, both retrieving card-value and word-value is done via explicit dereference. Packaging only one up destroys the symmetry, and would force future readers to look up the impl on trying to follow the logic here.